ext/lexbor/php_lexbor.c

```diff
@@ -0,0 +1,58 @@
+/*
+   +----------------------------------------------------------------------+
+   | Copyright (c) The PHP Group                                          |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | https://www.php.net/license/3_01.txt                                 |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Authors: Niels Dossche <nielsdos@php.net>                            |
+   |          Mate Kocsis <kocsismate@php.net>                            |
+   +----------------------------------------------------------------------+
+*/
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "php.h"
+#include "zend_globals.h"
+#include "ext/standard/info.h"
+
+#ifdef HAVE_LEXBOR
+
+#include "php_lexbor.h"
+
+/* Dynamically loadable module stuff */
+#ifdef COMPILE_DL_LEXBOR
+#ifdef ZTS
+ZEND_TSRMLS_CACHE_DEFINE()
+#endif
+ZEND_GET_MODULE(lexbor)
+#endif /* COMPILE_DL_LEXBOR */
+
+static PHP_MINFO_FUNCTION(lexbor)
+{
+	php_info_print_table_start();
+	php_info_print_table_row(2, "Lexbor support", "active");
```

Comment:
Would it make sense / would it be possible to print the Lexbor version here?

Reply:
I was considering this, but  Lexbor has a main version (2.5.0 is the one currently in development), and seperate versions for each component, so I wasn't sure what we want. @nielsdos Do you have any preference for exposing the version information for Lexbor?

Reply:
No real preference. Exposing the main lexbor version is fine by me.

Reply:
I implemented a quick and dirty solution to expose the information, since it changes only ~ once per year, so my assumption is that it's not going to be difficult to maintain the version number in the makefiles.

Reply:
We could also list all of them. PHPInfo is for human rather than programmatic consumption.

Reply:
As I see the question, it doesn't really matter if the granular version information is also exposed. At least I'm totally fine with the current phpinfo content.